### ISSUE TITLE
migration/upgrade from 4.1 to 5.0 fails

### ISSUE THREAD

**Issue by bestrocker221** [goauthentik/authentik#5650]
**Describe the bug**
docker run with newer version fails

**To Reproduce**
Upgrade from 4.1 to 5.0

**Expected behavior**
Working 

**Logs**
```
gauthentik-server-1      | Traceback (most recent call last):
gauthentik-server-1      |   File "<frozen runpy>", line 198, in _run_module_as_main
gauthentik-server-1      |   File "<frozen runpy>", line 88, in _run_code
gauthentik-server-1      |   File "/lifecycle/migrate.py", line 95, in <module>
gauthentik-server-1      |     execute_from_command_line(["", "migrate"])
gauthentik-server-1      |   File "/usr/local/lib/python3.11/site-packages/django/core/management/__init__.py", line 446, in execute_from_command_line
gauthentik-server-1      |     utility.execute()
gauthentik-server-1      |   File "/usr/local/lib/python3.11/site-packages/django/core/management/__init__.py", line 440, in execute
gauthentik-server-1      |     self.fetch_command(subcommand).run_from_argv(self.argv)
gauthentik-server-1      |   File "/usr/local/lib/python3.11/site-packages/django/core/management/base.py", line 402, in run_from_argv
gauthentik-server-1      |     self.execute(*args, **cmd_options)
gauthentik-server-1      |   File "/usr/local/lib/python3.11/site-packages/django/core/management/base.py", line 448, in execute
gauthentik-server-1      |     output = self.handle(*args, **options)
gauthentik-server-1      |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
gauthentik-server-1      |   File "/usr/local/lib/python3.11/site-packages/django/core/management/base.py", line 96, in wrapped
gauthentik-server-1      |     res = handle_func(*args, **kwargs)
gauthentik-server-1      |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
gauthentik-server-1      |   File "/usr/local/lib/python3.11/site-packages/django/core/management/commands/migrate.py", line 349, in handle
gauthentik-server-1      |     post_migrate_state = executor.migrate(
gauthentik-server-1      |                          ^^^^^^^^^^^^^^^^^
gauthentik-server-1      |   File "/usr/local/lib/python3.11/site-packages/django/db/migrations/executor.py", line 135, in migrate
gauthentik-server-1      |     state = self._migrate_all_forwards(
gauthentik-server-1      |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
gauthentik-server-1      |   File "/usr/local/lib/python3.11/site-packages/django/db/migrations/executor.py", line 167, in _migrate_all_forwards
gauthentik-server-1      |     state = self.apply_migration(
gauthentik-server-1      |             ^^^^^^^^^^^^^^^^^^^^^
gauthentik-server-1      |   File "/usr/local/lib/python3.11/site-packages/django/db/migrations/executor.py", line 252, in apply_migration
gauthentik-server-1      |     state = migration.apply(state, schema_editor)
gauthentik-server-1      |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
gauthentik-server-1      |   File "/usr/local/lib/python3.11/site-packages/django/db/migrations/migration.py", line 130, in apply
gauthentik-server-1      |     operation.database_forwards(
gauthentik-server-1      |   File "/usr/local/lib/python3.11/site-packages/django/db/migrations/operations/fields.py", line 235, in database_forwards
gauthentik-server-1      |     schema_editor.alter_field(from_model, from_field, to_field)
gauthentik-server-1      |   File "/usr/local/lib/python3.11/site-packages/django/db/backends/base/schema.py", line 788, in alter_field
gauthentik-server-1      |     self._alter_field(
gauthentik-server-1      |   File "/usr/local/lib/python3.11/site-packages/django/db/backends/postgresql/schema.py", line 245, in _alter_field
gauthentik-server-1      |     super()._alter_field(
gauthentik-server-1      |   File "/usr/local/lib/python3.11/site-packages/django/db/backends/base/schema.py", line 1045, in _alter_field
gauthentik-server-1      |     self.execute(self._create_unique_sql(model, [new_field]))
gauthentik-server-1      |   File "/usr/local/lib/python3.11/site-packages/django/db/backends/base/schema.py", line 199, in execute
gauthentik-server-1      |     cursor.execute(sql, params)
gauthentik-server-1      |   File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 67, in execute
gauthentik-server-1      |     return self._execute_with_wrappers(
gauthentik-server-1      |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
gauthentik-server-1      |   File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
gauthentik-server-1      |     return executor(sql, params, many, context)
gauthentik-server-1      |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
gauthentik-server-1      |   File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 84, in _execute
gauthentik-server-1      |     with self.db.wrap_database_errors:
gauthentik-server-1      |   File "/usr/local/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
gauthentik-server-1      |     raise dj_exc_value.with_traceback(traceback) from exc_value
gauthentik-server-1      |   File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
gauthentik-server-1      |     return self.cursor.execute(sql, params)
gauthentik-server-1      |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
gauthentik-server-1      |   File "/usr/local/lib/python3.11/site-packages/django_prometheus/db/common.py", line 69, in execute
gauthentik-server-1      |     return super().execute(*args, **kwargs)
gauthentik-server-1      |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
gauthentik-server-1      | django.db.utils.IntegrityError: could not create unique index "authentik_crypto_certificatekeypair_name_719603a4_uniq"
gauthentik-server-1      | DETAIL:  Key (name)=(authentik Self-signed Certificate_2) is duplicated.
```

**Comment by Changer90** [comment]
Hi, 
I had the same issue. When I set up authentik some weeks ago, it created the authentik self signed certificate 3 times. To update to 2023.5.0 I had to remove or rename the duplicate certificate names. I chose to rename them. I renamed the certificate keys in the database. After that, I was able to upgrade and everything is working now as expected.
Best regards
Changer

**Comment by mam** [comment]
I ran into exactly the same error upgrading from 2023.4.1 to 2023.5.0, on an instance I created about a week ago. I discovered there was 4x"authentik Self-signed Certificate". I decided to delete the three newest certificates, and seems to work fine now. Not sure if there are any caveats of deleting them(?)

**Comment by BeryJu** [comment]
This is fixed by #5637 and will be included in the next patch release, for now you can use the beta images (https://goauthentik.io/docs/installation/beta) which already have this fix

**Comment by BeryJu** [comment]
this is fixed by 2023.5.1
